build_ios/Pods/Target Support Files/Pods-Play!/GeneralSettings.xcconfig

```diff
@@ -9,13 +9,13 @@ ENABLE_BITCODE = NO
 CODE_SIGN_IDENTITY = -
 
 OTHER_CFLAGS = -fvisibility=hidden $(inherited) -isystem "${PODS_ROOT}/Headers/Public" -isystem "${PODS_ROOT}/Headers/Public/SDWebImage"
-OTHER_LDFLAGS = $(inherited) -ObjC -l"SDWebImage" -framework "ImageIO" -framework boost
```

Comment:
-framework boost should not have been removed from the submission version of this commit